frontend/src/utils/storage.ts

```
@@ -0,0 +1,399 @@
+      return [];
+    }
+
+    try {
+      const keys = [];
+      const prefixLength = this.prefix.length;
+
+      for (let i = 0; i < this.storage.length; i++) {
+        const key = this.storage.key(i);
+        if (key) {
+          if (this.prefix) {
+            if (key.startsWith(this.prefix + '_')) {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)